docs/learn-account-generation.md

```diff
@@ -219,6 +219,34 @@ with the password.
 
 ![pjs-05](assets/accounts/polkadot_js_05.jpg)
 
+### Multi-signature Accounts
+
+Multi-signature accounts are accounts created from several standard accounts (or even other multi-sig accounts). For a full explanation, please see the [Accounts Explainer section on multi-sigs](learn-accounts#multi-signature-accounts).
+
+On the [Accounts](https://polkadot.js.org/apps/#/accounts) tab, click the `Multisig` button. Enter the threshold and add signatories. The threshold must be less than or equal to the number of signatories. The threshold indicates how many members must be in agreement for an extrinsic submission to be successful. Click `Create` when done. We'll use the accounts Alice, Bob, and Charlie here.
+
+![Multi-sig account creation](/img/accounts/create-multisig.png)
+
+This merely calculates the multi-signature's address and adds it to your UI. The account does not exist yet, and is subject to the same [Existential Deposit and Reaping](learn-accounts#existential-deposit-and-reaping) rules as regular accounts. 
+
+Suppose we funded it with some tokens, and now want to send from the multi-sig to another account. 
+
+![Sending from multi-sig account](/img/accounts/send-from-multi.png)
+
+The next step is to sign the transaction from every required participating account; in the above case, two out of three signatories must sign.
```

Comment:
```suggestion
The next step is to sign the transaction from with enough accounts to meet the threshold; in the above case, two out of three signatories must sign.
```

Reply:
"every required participating account" - I see what you mean but I can see this being confusing that there are specific accounts that need to participate.